test/test-server/connect_handler.go

```diff
@@ -298,6 +303,7 @@ func handleCustomPluginUploadUrl(t *testing.T) http.HandlerFunc {
 		if r.Method == http.MethodPost {
 			uploadUrl := connectcustompluginv1.ConnectV1PresignedUrl{
 				ContentFormat: PtrString("ZIP"),
+				Cloud:         PtrString("aws"),
 				UploadId:      PtrString("e53bb2e8-8de3-49fa-9fb1-4e3fd9a16b66"),
```

Comment:
Can we use the built-in function from the SDK, instead of defining our own function?
```suggestion
				ContentFormat: connectcustompluginv1.PtrString("ZIP"),
				Cloud:         connectcustompluginv1.PtrString("aws"),
				UploadId:      connectcustompluginv1.PtrString("e53bb2e8-8de3-49fa-9fb1-4e3fd9a16b66"),
```